gatsby/src/docs/performance-monitoring/distributed-tracing.mdx

```diff
@@ -0,0 +1,258 @@
+
+### Why Tracing?
+
+Applications typically consist of interconnected components, which are also called services. As an example, let's look at a modern web application, composed of the following components, separated by network boundaries:
+
+- Frontend (Single-Page Application)
+- Backend (REST API)
+- Task Queue
+- Database Server
+- Cron Job Scheduler
+
+Each of these components may be written in a different language on a different platform. Each can be instrumented individually using a Sentry SDK to capture error data or crash reports, but that instrumentation doesn't provide the full picture, as each piece is considered separately. Tracing allows you to tie all of the data together.
+
+In our example web application, tracing means being able to follow a request from the frontend to the backend and back, pulling in data from any background tasks or notification jobs that request creates. Not only does this allow you to correlate Sentry error reports, to see how an error in one service may have propagated to another, but it also allows you to gain stronger insights into which services may be having a negative impact on your application's overall performance.
+
+Before learning how to enable tracing in your application, it helps to understand a few key terms and how they relate to one another.
+
+### Traces, Transactions, and Spans
+
+A **trace** represents the record of the entire operation you want to measure or track - like page load, an instance of a user completing some action in your application, or a cron job in your backend. When a trace includes work in multiple services, such as those listed above, it's called a **distributed trace**, because the trace is distributed across those services.
+
+Each trace consists of one or more tree-like structures called **transactions**, the nodes of which are called **spans**. In most cases, each transaction represents a single instance of a service being called, and each span within that transaction represents that service performing a single unit of work, whether calling a function within that service or making a call to a different service. Here's an example trace, broken down into transactions and spans:
+
+[<img alt="Diagram illustrating how a trace is composed of multiple transactions, and each transaction is composed of multiple spans." integrity="sha256-ZYePpKmK0AziV7mMVk1z8rgCN0WDsA+LKWlQXwTyoO8=" crossorigin="anonymous" src="/assets/performance/diagram-transaction-trace-65878fa4a98ad00ce257b98c564d73f2b802374583b00f8b2969505f04f2a0ef.png">](/assets/performance/diagram-transaction-trace-65878fa4a98ad00ce257b98c564d73f2b802374583b00f8b2969505f04f2a0ef.png)
```

Comment:
something perhaps odd here?

Reply:
nope but will fix `{% asset %}` shortly which is whats causing this